# ==============================================================================
# 描述: TeamMulticast消息中，“敌方信息表格”的一行。
#      代表关于单个敌方目标的融合情报。
# ==============================================================================
int32 id                              # 敌方ID
geometry_msgs/Point position          # 已知的敌方位置
builtin_interfaces/Time timestamp     # 该情报的时间戳